teamplate.rb

```diff
@@ -18,11 +18,12 @@
 generate "bootstrap:install", "static"
 generate "bootstrap:layout" # auto overwrite
 
+# Figaro
 # Other gems
-%w( quiet_assets pg ).each { |name| gem name }
+%w( quiet_assets pg rails-erd figaro).each { |name| gem name }
```

Comment:
It's probably worth doing the generator bit (that installs `config/application.yml` and adds it to the `.gitignore`) as well, so that we don't have to remember to do that.